OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -572,6 +572,32 @@
     <Compile Include="UtilityCommands\CheckCodeStyle.cs" />
     <Compile Include="UtilityCommands\ReplayMetadataCommand.cs" />
     <Compile Include="Widgets\Logic\ReplayUtils.cs" />
+    <Compile Include="Widgets\BackgroundWidget.cs" />
+    <Compile Include="Widgets\ButtonWidget.cs" />
+    <Compile Include="Widgets\ChatDisplayWidget.cs" />
+    <Compile Include="Widgets\CheckboxWidget.cs" />
+    <Compile Include="Widgets\ClientTooltipRegionWidget.cs" />
+    <Compile Include="Widgets\ColorBlockWidget.cs" />
+    <Compile Include="Widgets\DropDownButtonWidget.cs" />
+    <Compile Include="Widgets\GridLayout.cs" />
+    <Compile Include="Widgets\HotkeyEntryWidget.cs" />
+    <Compile Include="Widgets\ImageWidget.cs" />
+    <Compile Include="Widgets\LabelWidget.cs" />
+    <Compile Include="Widgets\LineGraphWidget.cs" />
+    <Compile Include="Widgets\ListLayout.cs" />
+    <Compile Include="Widgets\MapPreviewWidget.cs" />
+    <Compile Include="Widgets\PasswordFieldWidget.cs" />
+    <Compile Include="Widgets\PerfGraphWidget.cs" />
+    <Compile Include="Widgets\ProgressBarWidget.cs" />
+    <Compile Include="Widgets\RGBASpriteWidget.cs" />
+    <Compile Include="Widgets\ScrollItemWidget.cs" />
+    <Compile Include="Widgets\ScrollPanelWidget.cs" />
+    <Compile Include="Widgets\SliderWidget.cs" />
+    <Compile Include="Widgets\SpriteWidget.cs" />
+    <Compile Include="Widgets\TextFieldWidget.cs" />
+    <Compile Include="Widgets\TooltipContainerWidget.cs" />
+    <Compile Include="Widgets\ViewportControllerWidget.cs" />
+    <Compile Include="Widgets\VqaPlayerWidget.cs" />
```

Comment:
Sadly MonoDevelop does not automatically sort those alphabetically. I did it manually in https://github.com/OpenRA/OpenRA/pull/7564.